opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -356,10 +356,6 @@ namespace detail {
         }
     }
 
```

Comment:
Please do not remove this whitespace. In this file all functions are separated by 5 blank lines to make it easy to spot functions even though many are taking multiple screen pages.